### ISSUE TITLE
Videos give an error after a few seconds of playing the video

### ISSUE THREAD

**Issue by Xammand** [Rehike/Rehike#589]
### Rehike version

0.8.1

### Operating system

Windows 10 Home 22H2

### PHP version

8.2.12

### What's going on?

Every time I try to play a video, it gives an error seemingly at a random time of the video, it's always close to the start so like 1 minute or a bit more or less. Trying to skip a part of the video after whatever time the video will give an error it will load but then give that error. Refreshing the page seems like it has the video play for a bit longer but it will still break. I have tested every player and they all do this.

I may be wrong, but is it because YouTube has a thing where it doesn't load the full video but loads pieces of the video while it plays so there is a bug where it cannot load another piece of the video so it just gives an error? 

Even the latest player doesn't work. It even has it's own issue where it's stuck in an infinite loading loop and will not play at all.

This error was from the 2014 player, it errored at 39 seconds.
![Screenshot_359](https://github.com/user-attachments/assets/d5ddfbad-82a8-404a-b063-c42aadcefd30)

I recently reinstalled Rehike and I don't think it's an issue with my install because I tried out that v3 extension a day ago and I had the same problem there.

**Comment by kawapure** [comment]
YouTube added an experiment for additional request attestation. We cannot fix this at the moment. This issue affects both Rehike and V3. Please try a different account or VPN, or use the latest player.

**Comment by Xammand** [comment]
> YouTube added an experiment for additional request attestation. We cannot fix this at the moment. This issue affects both Rehike and V3. Please try a different account or VPN, or use the latest player.

Signed out, videos work for me but like I said the latest player just goes into an infinite loading loop right away.

**Comment by kawapure** [comment]
Your only option at the moment is to watch videos signed out.

**Comment by Xammand** [comment]
Right. I hope this gets fixed soon. I have also just disabled Rehike to watch a video too.

**Comment by kawapure** [comment]
The V3 developer, Reprety, has discovered a potential workaround. It should be implemented into V3 and Rehike sometime soon, but I can't provide an ETA for I have no clue when I'll get the information either.
